airflow/models.py

```diff
@@ -331,6 +333,19 @@ def process_file(self, filepath, only_if_updated=True, safe_mode=True):
         if filepath is None or not os.path.isfile(filepath):
             return found_dags
 
+        # if pickled dag, we return directly after adding them to our DAG
+        pickle_found = True
+        timestamp = str(os.path.getmtime(filepath)).replace(".", "_")
+        try:
+            pickle_path = filepath[:-2]+timestamp+".p"
+            pickled_dags = pickle.load(open(pickle_path, "rb"))
+            for dag in pickled_dags:
+                self.dags[dag.dag_id] = dag
+            return pickled_dags
+        except:
```

Comment:
Broad exception.

`FileNotFound` should be catch (as it is expected) and others exception should at least be logged with `self.log.exceptions()`.